students/cfarris/lessons/lesson09/assignment/test_mdb_db.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env Python3
+
+'''
+Lesson09 Pytest for mdb_db.py
+'''
+
+from pymongo import MongoClient
+import pytest
+from mdb_db import (import_data,
+                    _read_csv,
+                    show_avail_products,
+                    show_rentals,
+                    check_redundancy)
+
+
+class MongoDBConnection():
+    """Connect to MongoDB"""
+
+    def __init__(self, host='127.0.0.1', port=27017):
+
+        self.host = host
+        self.port = port
+        self.connection = None
+
+    def __enter__(self):
+        self.connection = MongoClient(self.host, self.port)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.connection.close()
```

Comment:
You don't have to re-create this context manager in every file, you can just import it.